indexer/docs/README.md

```diff
@@ -0,0 +1,6 @@
+# @eth-optimism/indexer/docs
+
+The [vitepress](https://vitepress.dev/) based docs application for indexer
+
+These are not official docs and will be moved to [real docs](https://community.optimism.io) at the time of this indexer being GA to all OP stack chains
```

Comment:
Actually not planning on ever landing this as I don't want a package.json and pnpm-lock.yaml file to get merged